### PR TITLE
sidebar stylez

### PR DESCRIPTION
**Pull request by brendansudol**
This is a quick iteration on improving sidebar stylez (a la https://github.com/18F/cms-hitech-apd/issues/188#issuecomment-370439011)

It's not pixel perfect, but it's a lot better than it was.

Also removed `react-xray` (which wasn't proving to be useful although it was pretty).

Preview:
sidebar-iterate

### This pull request is ready to merge when...
- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This feature is done when...
- [ ] Design has approved the experience

### CODE REVIEW

File: web/src/containers/AuthChecker.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ class AuthChecker extends Component {
     const { authInit, children } = this.props;
 
     if (!authInit) return null;
-    return <div>{children}</div>;
+    return <Fragment>{children}</Fragment>;
```
Comment (brendansudol) [inline web/src/containers/AuthChecker.js]: the wrapping `div` here was unnecessary (and more importantly, was throwing off the full-width-ness of the sidebar)

File: web/src/containers/TopNav.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ class TopNav extends Component {
     const { authenticated } = this.props;
 
     return (
-      <nav className="clearfix py1 white bg-black">
+      <header className="clearfix py1 border-bottom border-silver">
```
Comment (brendansudol) [inline web/src/containers/TopNav.js]: `header` is better semantically here than `nav`

File: web/src/components/Container.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  <div className="site-main">
+    <div className="container p2">{children}</div>
+  </div>
+);
+
+Container.propTypes = {
+  children: PropTypes.node.isRequired
+};
+
+export default Container;
```
Comment (mgwalker) [inline web/src/components/Container.js]: What's the purpose of this `Container` element?  I see it's used by the login page, but nowhere else?

**Comment by nkkl** [comment]
@lauraponce are you good with this?

File: package-lock.json [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+      "resolved": "https://registry.npmjs.org/through/-/through-2.3.8.tgz",
+      "integrity": "sha1-DdTJ/6q8NXlgsbckEV1+Doai4fU="
+    },
+    "xtend": {
+      "version": "4.0.1",
+      "resolved": "https://registry.npmjs.org/xtend/-/xtend-4.0.1.tgz",
+      "integrity": "sha1-pcbVMr5lbiPbgg77lDofBJmNY68="
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (brendansudol) [reply package-lock.json]: oops, not sure where this came from -- will slash! ⚔️

**Comment by brendansudol** [comment]
one thing to note -- it's not perfect yet @lauraponce but hopefully directionally better!

File: web/src/components/Container.js (hunk 3 above)
Comment (brendansudol) [reply web/src/components/Container.js]: so `Container` should be used when there's no sidebar, so it's basically the one-column layout wrapper (which so far is only really the login page)

**Pull request merged**